frontend/src/components/Submissions/SubmissionsGrid.js

```diff
@@ -57,7 +58,7 @@ class SubmissionsGrid extends React.Component {
                     dropdownMenu={['filter_by_value', 'filter_action_bar']}
                     // make actions clickable
                     afterOnCellMouseDown={(event, coords, TD) => {
-                        if (coords.row !== -1 && coords.col !== -1) {
+                        if (coords.row !== -1 && coords.col !== -1 && event.button === 0) {
```

Comment:
what is `event.button === 0`?

Reply:
event.button indicates which button on the mouse was clicked, event.button === 0 is true when the left button on the mouse is clicked. Maybe it was added as a safety to make sure the user is purposely clicking something on the grid